### PR TITLE
Peppy.Osu! version 2021.828.0

### PR DESCRIPTION
**Pull request by aeongdesu**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26111)

### CODE REVIEW

File: manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://github.com/ppy/osu/releases/download/2021.828.0/install.exe
+  InstallerSha256: 874F751400592F4810EFAE4EB438A38C9BB4967F6C8F6B6E402F31A6972A6E18
+  InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: Peppy.Osu!
```
Comment (ItzLevvie) [inline manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml:20]:
```suggestion
    SilentWithProgress: Peppy.Osu!
```
Oops, I forgot to spot this from @OfficialEsco's PR but is `Peppy.Osu!` supposed to be here?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml:20]: Uhhh, no its not 🤦‍♂️ Guess the Powershell History messed me up

Comment (OfficialEsco) [inline manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml:20]:
```suggestion
    SilentWithProgress: /silent
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml (hunk 1 above)
Comment (KeepSOBP) [reply manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml]: 💀 seems wingetcreate got drunk

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**